### ISSUE TITLE
Issue in building logcourier.exe in windows 7

### ISSUE THREAD

**Issue by Rajeshkumar123** [driskell/log-courier#168]
![image](https://cloud.githubusercontent.com/assets/7466386/7855962/56051e5c-04da-11e5-9ea0-a9f556a99c80.png)

**Comment by driskell** [comment]
Please follow the instructions here:
https://github.com/driskell/log-courier/blob/master/README.md#windows

Use msysGit console, not cmd.exe. After installing msysGig there should be a Mingw32 link on the desktop or in start menu IIRC. Use that.

**Comment by driskell** [comment]
I will update documentation on the console to use as it's not clear at the moment. Apologies for any confusion.

**Comment by Rajeshkumar123** [comment]
I have used msysGit. There is no output for me. The screen remains idle.

![image](https://cloud.githubusercontent.com/assets/7466386/7856367/5336d0c8-04dd-11e5-8b49-70188ccdb370.png)

**Comment by Rajeshkumar123** [comment]
Do I need to give any input after make . I am pointing the log-courier directory only.

**Comment by driskell** [comment]
Rather than snapshots could you paste text? Search engines or GitHub search can't index images so it'll make it harder for people to get here if they have similar issue or even a resolution.

What if you run `go version`? It seems odd that you've placed the log-courier folder in `/c/go` as only the Golang installation should be there. Are you sure you first installed Golang to `/c/go`? Can you try cloning log-courier into another directory?

**Comment by driskell** [comment]
Can you open mingw32 and just follow the commands from the instructions exactly? (Use `/c/go` rather than `/c/Go` though if that's where you installed Golang)

https://github.com/driskell/log-courier/blob/master/README.md#windows

**Comment by Rajeshkumar123** [comment]
I thought pasting image will give you clear idea. Hereafter I will post the text. My go version is "go version go1.4.2 windows/386"

I executed the step following the step mentioned in the document . I am facing below issues.

156687@RAJESH ~
$ export GOROOT=/c/Go

156687@RAJESH ~
$ export PATH=$PATH:$GOROOT/bin

156687@RAJESH ~
$ cd /C/Java1/logstashfiles/log-courier-master

156687@RAJESH /C/Java1/logstashfiles/log-courier-master
$ make
build/fix_version ""
Set Log Courier Version
build/setup_root
GOPATH: c:/Java1/logstashfiles/log-courier-master
go get -d -tags "" log-courier
go install -tags "" log-courier
# log-courier
src\log-courier\log-courier.go:76: undefined: core.Log_Courier_Version
src\log-courier\log-courier.go:177: undefined: core.Log_Courier_Version
make: *** [bin/log-courier] Error 2

**Comment by driskell** [comment]
I take it from #169 you managed to build OK? What was the problem and how did you fix it? Just for future reference, thanks!

**Comment by Rajeshkumar123** [comment]
I restarted and run the process by removing the log-courier folder from go file and executed it it creates the exe.

**Comment by Rajeshkumar123** [comment]
I am facing the issue which I mentioned in #169 . Why log courier fails to fetch the file.

**Comment by driskell** [comment]
> I restarted and run the process by removing the log-courier folder from go file and executed it it creates the exe.

Could you rephrase that? I didn't understand it, sorry.

**Comment by Rajeshkumar123** [comment]
I pasted the log-courier folder inside c/go and I tried to build it. I removed the folder and from c/go and it is working fine now.

**Comment by driskell** [comment]
OK thanks. Not sure myself what you mean but maybe others with the issue will know what to do! Thanks for dropping the note.
